process_report/process_report.py

```diff
@@ -335,6 +366,24 @@ def validate_pi_names(dataframe):
     return dataframe
 
 
+def validate_pi_aliases(dataframe: pandas.DataFrame, alias_dict: dict):
+    for pi, pi_aliases in alias_dict.items():
+        for pi_alias in pi_aliases:
+            if not dataframe.loc[dataframe[PI_FIELD] == pi_alias].empty:
+                print(f"Warning: Found alias {pi_alias} for {pi}")
+
+        dataframe.loc[dataframe[PI_FIELD].isin(pi_aliases), PI_FIELD] = pi
```

Comment:
The entire point in using the vector operations provided by pandas is to avoid performing iterations in Python. Just get rid of the inner for loop.

```suggestion
    for pi, pi_aliases in alias_dict.items():
        dataframe.loc[dataframe[PI_FIELD].isin(pi_aliases), PI_FIELD] = pi
```

Reply:
I kept the inner loop because I wanted to log whether a PI's alias was encountered. I don't think there is a way to do this without using iterating through the PI's aliases?

Reply:
It's okay not to log when alias encountered.